app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -616,5 +663,75 @@ private PlayQueue getPlayQueue(final int index) {
         }
         return new SinglePlayQueue(streamInfoItems, index);
     }
+
+    private class RemoveWatchedStreams{
```

Comment:
I'm not sure this should be done inside a nested class. You could do the same things in three functions without nesting things (at least, that's how I have seen it done in other places in the repository)

Reply:
Removed the class in commit [https://github.com/TeamNewPipe/NewPipe/pull/3065/commits/81bcac5d2800b14a4e1917c9ffbfe4064b5fb967](https://github.com/TeamNewPipe/NewPipe/pull/3065/commits/81bcac5d2800b14a4e1917c9ffbfe4064b5fb967)